lahja/asyncio/endpoint.py

```diff
@@ -125,61 +125,28 @@ def __init__(self, reader: StreamReader, writer: StreamWriter) -> None:
             raise RemoteDisconnected()
 
 
-class RemoteEndpoint:
-    """
-    Represents a connection to another endpoint.  Connections *can* be
-    bi-directional with messages flowing in either direction.
-
-    A 'message' can be any of:
-
-    - ``SubscriptionsUpdated``
-            broadcasting the subscriptions that the endpoint on the other side
-            of this connection is interested in.
-    - ``SubscriptionsAck``
-            acknowledgedment of a ``SubscriptionsUpdated``
-    - ``Broadcast``
-            an event meant to be processed by the endpoint.
-    """
-
-    logger = logging.getLogger("lahja.endpoint.asyncio.RemoteEndpoint")
-
+class RemoteEndpoint(BaseRemoteEndpoint):
```

Comment:
The goal behind breaking this up might be more clear if it was called `AsyncioRemoteEndpoint`.